### PR TITLE
Add Symfony 7 support

### PR DESCRIPTION
**Pull request by thecrimpmaster**

### CODE REVIEW

**Comment by mkilmanas** [comment]
Hi @thecrimpmaster - could you please address the static analysis issues?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ jobs:
                 run: bin/phpspec run --format=dot
 
             -   name: Run tests (phpunit)
-                run: ./vendor/bin/phpunit -v
+                run: ./vendor/bin/phpunit
```
Comment (thecrimpmaster) [inline .github/workflows/build.yml:77]: Verbose option has been removed in version 10 (https://github.com/sebastianbergmann/phpunit/blob/10.0.0/ChangeLog-10.0.md) 

Is this ok to just remove it?

**Comment by thecrimpmaster** [comment]
Sorry, I just kind of created, pushed, and didn't check back due to time constraints. 

Made some changes and all builds passing now. Probably worth a review or two, as not sure about all changes

File: features/formatter/developer_is_shown_diffs.feature [hunk 2]
```diff
@@ -193,11 +193,17 @@ Feature: Developer is shown diffs
     Then I should see:
       """
             -    'i' => 1
+      """
+    And I should see:
```
Comment (thecrimpmaster) [inline features/formatter/developer_is_shown_diffs.feature:197]: There seems to be an "," added to the diff message in php 8. I've split the expectations out to pass in 7 and 8. Feels slightly clunky. Love to hear others' opinions

**Comment by ciaranmcnulty** [comment]
Thanks @thecrimpmaster !

**Pull request merged**